public/static/js/projects.js

```diff
@@ -387,7 +398,7 @@ mciModule.controller('ProjectCtrl', function($scope, $window, $http, $location,
       return d.changed;
     });
 
-    $scope.settingsFormData.project_aliases = $scope.github_aliases.concat($scope.patch_aliases);
+    $scope.settingsFormData.project_aliases = $scope.github_aliases.concat($scope.patch_aliases, $scope.commit_queue_aliases);
```

Comment:
Concating them in this way leads to an incorrect error message "Couldn't save project: must specify exactly one of task regex or tags on line #3" because the line numbers are not meaningful

Reply:
fixed the order they get concatenated

Reply:
The problem isn't the order - it's that you're concatenating 2 fields that are separate in the UI. I would change the validation to tell the user which alias is invalid

Reply:
should be good now: I separated the alias types sent to the server so the server can differentiate.